src/search/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './types';
+export * from './SearchCard';
+export * from './SearchPopup';
```

Comment:
I suggest putting all the editor's interface stuff in the modules folder. Let's move the search folder there.